### ISSUE TITLE
404 getting image from page bundle in post URL ending in .html

### ISSUE THREAD

**Issue by rasmusson** [chipzoller/hugo-clarity#337]
**I have checked all the prerequisites below and I'm yet experiencing a problem**

- [x] Read the README.md
- [x] Have the extended version of Hugo installed
- [x] Used the exampleSite's config.toml as a reference
- [ ] I have included my repository link. Instead of pasting long snippets here.
My source is currently not public

**Describe the bug**
Because of migration from a old blog I have have used the front matter parameter URL to set the URL of the post.
When I set the URL to be a path ending in ".html" the thumbnail and feature imaged can not be fetched.
The image src is pointing to "/11/verifying-signatures-with-opensaml.html/opensaml4-signing.jpg", while the image is actually directly under "/11" as a sibling to verifying-signatures-with-opensaml.html.
If I remove the .html ending the images resolves fine.

> Avoid long-winded descriptions

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: Chrome 102

**Comment by rootwork** [comment]
Hmm. Yes that makes sense what's going on, I'm not sure how to resolve it, though. What happens if you set the path to the image explicitly as `/11/opensaml4-signing.jpg`? The leading slash might tell Hugo to use that path directly rather than construct a relative URL.

**Comment by rasmusson** [comment]
It still looking for it relatively, path is now 2012/11/verifying-signatures-with-opensaml.html/2012/11/opensaml4-signing.jpg

**Comment by rasmusson** [comment]
Right. Maby use in to check for presens of . as a substring of the path? 
{{ if in path "." }}

**Comment by rasmusson** [comment]
No, sorry there can be other dots in there. Perhaps a regex using findRE?
